src/client/cassh

```diff
@@ -337,23 +337,23 @@ if __name__ == '__main__':
         print('realname = ursula.ser@domain.fr')
         exit(1)
 
-    CLIENT = CASSH(read_conf(CONF_FILE))
+    LBC = CASSH(read_conf(CONF_FILE))
```

Comment:
Why do you change that var name ? This is an open source project, you should try not to use anything "company branded"

Reply:
:+1:  with @vdemonchy